src/views/Wallet/Create/PassPhrase.vue

```diff
@@ -94,7 +94,7 @@ export default {
     },
 
     containsSpecials () {
-      return !!this.passphrase.match(/[!@#$%^&*()_+-=\\[\]{};':"\\|,.<>/?]/g)
+      return !!this.passphrase.match(/[!@#$%^&*()_+\-=[\]{};':"\\|,.<>/?]+/g)
```

Comment:
This regex differs from the regex used on the iOS wallet, making the two incompatible. That's highly unwanted.
In another commit I've changed the label and thus the pass phrase requirements. I'll revert this change